### DISCUSSION TITLE
how to hide placeholder

### DISCUSSION THREAD

**Discussion by nickcom** [rsuite/rsuite#1870] [Q&A] [answered]
the placeholder component is very handy. What is the suggested way to hide it once the expected component is loaded?

**Comment by SevenOutman** [comment, marked as answer]
You could simply use a boolean state to render it conditionally. Set the state to `true` initially and `false` once your component is loaded.

```jsx
<>
  {showPlaceholder && (
    <Placeholder />
  )}
</>
```
